lms/static/sass/course/_info.scss

```diff
@@ -1,11 +1,37 @@
+div.welcome-wrapper {
+  max-width: 1180px;
+  margin: 0 auto;
+
+  .welcome {
+    margin: $baseline;
+    h1 {
+      margin-bottom: 5px;
+      color: $dark-gray1;
+      font-size: 24px;
+    }
+    h2 {
```

Comment:
For both the title and subtitle I would add padding similar to the Teams header. (that being said if we use the page-header / page-subtitle classes we might actually be able to rely on some of the existing styles, unless the teams header styling is scoped to .teams-header specifically.